FrontEnd/styles.css

```diff
@@ -1,10 +1,32 @@
 body{
     margin: 0;
+    font-size: 14px;
 }
 .container{
     width: 800px;
     margin: auto;
 }
+.container::before{
```

Comment:
We can have more than 1 container in the app, So I think we should change the name to more specifically